nvflare/fuel/f3/sfm/conn_manager.py

```diff
@@ -206,7 +206,7 @@ def send_message(self, endpoint: Endpoint, app_id: int, headers: Headers, payloa
 
     def register_message_receiver(self, app_id: int, receiver: MessageReceiver):
         if self.receivers.get(app_id):
-            raise CommError(CommError.BAD_DATA, f"Receiver for app {app_id} is already registered")
+            raise CommError(CommError.BAD_CONFIG, f"Receiver for app {app_id} is already registered")
```

Comment:
When should we use `CommError.BAD_CONFIG` VS `CommError.BAD_DATA`?

Reply:
BAD_CONFIG is for errors happened locally and BAD_DATA is for errors in the data received from remote endpoint.